app/controllers/targets_controller.rb

```diff
@@ -36,4 +36,17 @@ def target_params
   def topics
     @topics ||= Target::TOPICS
   end
+
+  def edit
+    @target = Target.find(params[:id])
```

Comment:
Here and in the destroy method, you can define and use a `helper_method` for target